code/modules/reagents/reagent_containers/iv_bag.dm

```diff
@@ -202,5 +197,8 @@
 	blood_type = "O-"
 
 /obj/item/reagent_containers/iv_bag/slime
-	name = "\improper IV Bag (Slime Jelly)"
 	list_reagents = list("slimejelly" = 200)
+
+/obj/item/reagent_containers/iv_bag/salglu/Initialize(mapload)
```

Comment:
```suggestion
/obj/item/reagent_containers/iv_bag/slime/Initialize(mapload)
```

Reply:
I feel so bad about this right now.. (Fixed)